config/config.go

```diff
@@ -234,7 +234,7 @@ type StateTriesConfig struct {
 type TrieStorageManagerConfig struct {
 	PruningBufferLen   uint32
 	SnapshotsBufferLen uint32
-	MaxSnapshots       uint8
+	MaxSnapshots       uint32
```

Comment:
could this cause comapatibility issues?

Reply:
No, this is strictly a processing component. The max snapshot value is strictly used when deciding to keep (or not) an older trie snapshot. This is saved nowhere.